src/gridstack.ts

```diff
@@ -2251,8 +2251,8 @@ export class GridStack {
         if (wasAdded) {
           this.engine.cleanupNode(node); // removes all internal _xyz values
           node.grid = this;
+          delete node.grid._isTemp;
         }
-        delete node.grid._isTemp;

```

Comment:
To reproduce the issue:

1. Enable dragging a widget from outside onto the grid.
2. Ensure the grid is crowded, leaving very little space (less than the size of the widget).
3. Drag the widget into that small space.

This will cause the addition of the widget to fail and become unresponsive. If you need further assistance, feel free to reach out. Thank you!